aviary/docs/user_guide/reserve_missions.md

```diff
@@ -5,40 +5,65 @@
 
+### Caveats when using 2DOF
+
+If you are using 2DOF equations of motion (EOM) in your problem (i.e. `settings:equations_of_motion,2DOF`) there are some additional things you need to be aware of.
+The name of the reserve phase should include one of the keywords to indicate which EOM from 2DOF will be selected.
+Valid keywords include: `accel`, `ascent`, `climb`, `cruise`, `desc`.
+This is because 2DOF uses different EOMs for different phases and we need to let `methods_for_level2.py` know which method to select.
+This is why in the example in the first paragraph above, the phase was named `reserve_cruise`.
+Do not worry about phase naming if you are using Height-Energy EOM as all those EOMs are the same for every phase.
+
 ## Theory
 
 When adding a reserve phase, `check_and_preprocess_inputs()` divides all the phases into two dictionaries: `regular_phases` which contain your nominal phases and `reserve_phases` which contains any phases with the `reserve` flag set to `True`.
+Additionally, `check_and_preprocess_inputs()` will add the `"analytic"` flag to each phase.
+This is used to indicate if a phase is an analytic phase (i.e. Breguet range) or a ordinary differential equation (ODE).
 
 Only the final mission mass and range from `regular_phases` are automatically connected to the first point of the `reserve_phases`.
-Altitude and other state variables are not automatically connected, allowing you to start the reserve mission at whatever altitude you want.
+All other state variables (i.e. altitude, mach) are not automatically connected, allowing you to start the reserve mission at whatever altitude you want.
+
+The `"analytic"` flag helps to properly connect phases for 2DOF missions. 
+2DOF `cruise` missions are analytic because they use a Breguet range calculation instead of integrating an EOM. 
+Analytic phases have a slightly different naming convention in order to access state/timeseries variables like distance, mass, and range compared with their non-analytic counterparts.
 
-Setting `optimize_mach` or `optimize_altitude` to `True` on regular or reserve phases will work.
-To be clear, the Mach and altitude values (even if set to be optimized) will not be connected between regular and reserve phases.
+You cannot create a reserve mission that enforces time or range constraints over multiple phases (i.e specify the total range covered by a climb + cruise + descent).
+This is because each constraint `"target_distance"` or `"target_time"` is only enforced on a single phase.
 
 It is essential that you run `prob.check_and_preprocess_inputs()` after `prob.load_inputs()` to make sure that regular and reserve phases are separated via `phase_separator()`.
 
+### Advanced Users and Target Duration Phases 
+For advanced users, instead of just copying a phase you used before, you might completely specify a new phase from scratch. 
```

Comment:
Given this, I wondering if maybe it doesn't make sense to show the examples where we copy cruise and change a few options. It is probably cleaner to just copy the whole phase_info, and then add the new phase to it from scratch. In particular, the analytic cruise reserve phase only has a few values that need to be set.

Reply:
there should be at least one example each way (copy existing phase and define entire phase)

Reply:
The first two examples (fixedtime and fixedrange) both show copying phases. The last example (multiphase) shows specifying new phases from scratch.